lbrynet/lbrynet_daemon/Daemon.py

```diff
@@ -1562,17 +1589,23 @@ def _start_file(f):
 
     def jsonrpc_get_est_cost(self, p):
         """
-        Get estimated cost for a lbry uri
+        Get estimated cost for a lbry stream
 
         Args:
             'name': lbry uri
+            'size': stream size, in bytes. if provided an sd blob won't be downloaded.
         Returns:
             estimated cost
         """
 
-        name = p[FileID.NAME]
+        if 'size' in p:
+            size = p['size']
```

Comment:
As a general rule, when you have a new indented block, think about making it a function.

```
if 'size' in p:
    return self._render_est_cost_from_stream_size(p)
```